external-api/src/http/external_match.rs

```diff
@@ -177,3 +180,55 @@ impl From<ExternalMatchResult> for ApiExternalMatchResult {
         }
     }
 }
+
+/// A signed quote for an external order
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct SignedExternalQuote {
+    /// The quote
+    pub quote: ApiExternalQuote,
+    /// The signature
+    pub signature: String,
+}
+
+/// A quote for an external order
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct ApiExternalQuote {
+    /// The mint of the quote token in the matched asset pair
+    pub quote_mint: String,
+    /// The mint of the base token in the matched asset pair
+    pub base_mint: String,
+    /// The amount of the quote token exchanged by the match
+    pub quote_amount: Amount,
+    /// The amount of the base token exchanged by the match
+    pub base_amount: Amount,
+    /// The direction of the match
+    pub direction: OrderSide,
+    /// The price of the match
+    pub price: f64,
+    /// The timestamp of the quote
+    pub timestamp: u64,
```

Comment:
not sure what the intended consumption of this field is, but just flagging that we could make it the timestamp from the timestamped price used by the matching engine, if we pass that back up

Reply:
to make sure the quote is not stale, I'd prefer to set it more directly as the time at which the quote was issued.